nekoyume/Assets/StreamingAssets/Localization/common.csv

```diff
@@ -1080,3 +1080,8 @@ UI_GRINDING_CRYSTALMAIL_FORMAT,I’ve finished grinding euqipment items & earned
 UI_GRINDING_NCGCRYSTALMAIL_FORMAT,I’ve finished grinding equipment items & earned {0}, {1}.,분해를 마치고 {0}, {1}을 얻었습니다.,,,,,,,,,,,,,,,,
 UI_MONSTERCOLLECTION_UPDATED,Monster collection state has updated.,몬스터 콜렉션 정보가 갱신되었습니다.,,,,,,,,,,,,,,,,
 UI_GRIND_NPC_QUOTE,Start grinding,분해 시작,,,,,,,,,,,,,,,,
+UI_GRIND_APREFILLGRIND_FORMAT,"Not enough action points.
+Do you want to recharge your action points by AP Potion and do Grind?
+(Count of AP Potion : {0})","AP가 부족합니다.
+AP 포션을 사용하여 AP를 채우고 분해를 진행하시겠습니까?
+(AP 포션 보유량 : {0})",,,,,,,,,,,,,,,,
```

Comment:
I think this format can be split more like this.
```suggestion
UI_APREFILL_GUIDE_FORMAT,"Not enough action points.
Do you want to recharge your action points by AP Potion and {0}?
(Count of AP Potion : {1})","AP가 부족합니다.
AP 포션을 사용하여 AP를 채우고 {0}를 진행하시겠습니까?
(AP 포션 보유량 : {1})",,,,,,,,,,,,,,,,
```

Or making it can be used generally.
```suggestion
UI_APREFILL_GUIDE_FORMAT,"Not enough action points.
Do you want to recharge your action points by AP Potion?
(Count of AP Potion : {1})","AP가 부족합니다.
AP 포션을 사용하여 AP를 채우시겠습니까?
(AP 포션 보유량 : {1})",,,,,,,,,,,,,,,,
```